conf/app.routes

```diff
@@ -1,12 +1,13 @@
-GET     /api/teams/:team/slug-dependencies     @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.dependenciesOfSlugForTeam(team, flag ?= "latest")
-GET     /api/repository-name                   @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.repositoryName(group: String, artefact: String, version: String)
-GET     /api/module-dependencies/:repository   @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.moduleDependencies(repository: String, version: Option[String])
-GET     /api/latest-version                    @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.latestVersion(group: String, artefact: String)
-GET     /api/meta-artefact/:repository         @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.metaArtefact(repository: String, version: Option[String])
+GET     /api/sluginfo                                      @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.slugInfo(name: String, version: Option[String])
+GET     /api/teams/:team/dependencies                      @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.dependenciesForTeam(team)
+GET     /api/serviceDeps                                   @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.getServicesWithDependency(flag: String ?= "latest", group: String, artefact: String, versionRange: String, scope: Option[List[String]] ?= None)
+GET     /api/groupArtefacts                                @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.getGroupArtefacts
+GET     /api/jdkVersions                                   @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.findJDKForEnvironment(team: Option[String], flag: String ?= "latest")
+GET     /api/bobbyViolations                               @uk.gov.hmrc.servicedependencies.controller.BobbyRuleViolationController.findBobbyRuleViolations
+GET     /api/historicBobbyViolations                       @uk.gov.hmrc.servicedependencies.controller.BobbyRuleViolationController.findHistoricBobbyRuleViolations(query: List[BobbyRuleQuery], from: java.time.LocalDate, to: java.time.LocalDate)
+GET     /api/teams/:team/slug-dependencies                 @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.dependenciesOfSlugForTeam(team, flag ?= "latest")
+GET     /api/repository-name                               @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.repositoryName(group: String, artefact: String, version: String)
+GET     /api/module-dependencies/:repository/allVersions   @uk.gov.hmrc.servicedependencies.controller.ServiceDependenciesController.moduleDependenciesAllVersions(repository: String)
```

Comment:
The URL is a bit strange for REST - since there is no resource type or property called `allVersions` (it looks like a projection into the result of `/api/module-dependencies/:repository`.

I think we should review this, even if it means renaming `/api/module-dependencies/:repository` to make it intuitive